packages/framework/fluid-framework/src/index.ts

```diff
@@ -152,7 +152,7 @@ export type {
 	ISharedSegmentSequence,
 } from "@fluidframework/sequence/internal";
 
-export {
+export type {
```

Comment:
why this change? seems like `SharedString` still needs to be usable as a value in public api, e.g. in dynamic DDS creation:

```const myNewString = await container.create(SharedString)```

Reply:
interesting. i don't think i made an direct modifications to this file, maybe an auto-formatter bug. i'll revert.

Reply:
![image](https://github.com/user-attachments/assets/9259fe09-3984-4d85-91dd-7b5756b712e2)

Reply:
this breaks the build. the error doesn't actually call out SharedString. I'll split the import to fix, but odd the autofix does the wrong thing. FYI @Josmithr @tylerbutler

Reply:
i also don't think we type test fluid-framework, so nothing catches this.

Reply:
> this breaks the build. the error doesn't actually call out SharedString. I'll split the import to fix, but odd the autofix does the wrong thing. FYI @Josmithr @tylerbutler

I don't think I've seen the auto-fixer change an export that was not type-only to a type-export before. If it's doing that, we may want to disable auto-fix for that rule.

> i also don't think we type test fluid-framework, so nothing catches this.

This seems like a pretty serious oversight in our tooling. We should schedule work for this.

Reply:
@Josmithr do you know the right place to file a bug like this, and which team would own it? if you do would you mind filing it, and linking it here?

Reply:
Ownership of some of this tooling stuff remains pretty ambiguous (something my team has been pushing on in retrospectives). I'm happy to take on the eslint update (I have another update I want to make in that space anyways). https://dev.azure.com/fluidframework/internal/_workitems/edit/22618

I think Team Sasha/Jason has been owning the type-test infra? Is that right? @jason-ha

Reply:
In the past we haven't worried about type testing fluid-framework since it just re-exports APIs that are type-tested in their "real" packages. But that was a long time ago, and we have since stopped using `export *` so messing up the exports is more likely now. It was also based on the fact that the entire API surface was type-tested, though, and that's no longer the case for most packages which only type-test a subset of their API. It should still be a super-set of what's exposed in fluid-framework, though.